### PR TITLE
Added Mergesort in c programming folder

### PR DESCRIPTION
**Pull request by Vybhavigithub**
solved #77 
please do check

### CODE REVIEW

File: C-ProgramQuicksort/Quicksort.c [hunk 1]
```diff
@@ -0,0 +1,104 @@
+# QUICKSORT IN  C PROGRAMMING :
```
Comment (Abishethvarman) [inline C-ProgramQuicksort/Quicksort.c:1]: this file seems good. Delete except this file

File: C-ProgramQuicksort/Quicksort.c
Comment (Abishethvarman) [inline C-ProgramQuicksort/Quicksort.c:1]: Check the file path. place it in the correct folder

**Review by Abishethvarman — CHANGES_REQUESTED** [review]

**Comment by Abishethvarman** [comment]
> solved #77 please do check

Add proper description

**Comment by Abishethvarman** [comment]
@Vybhavigithub rename the PR title

**Comment by Vybhavigithub** [comment]
> @Vybhavigithub rename the PR title

okay

**Pull request closed without merging**